doc/kernel/other/interrupts.rst

```diff
@@ -389,6 +412,13 @@ creates an _irq_to_interrupt_vector array which maps an IRQ line to its
 configured vector in the IDT. This is used at runtime by :c:macro:`IRQ_CONNECT`
 to program the IRQ-to-vector association in the interrupt controller.
 
+For dynamic interrupts, it is necessary for the build to generate some 4-byte
```

Comment:
How about,

    For dynamic interrupts, the build must generate 4-byte ...

Reply:
@dbkinder can you re-visit?